source/_integrations/analytics.markdown

```diff
@@ -18,9 +18,9 @@ Home Assistant allows users to share their usage data via the analytics integrat
 
 ## Data Collection
 
-The information sent depends on what options you opt-in to. You can opt-in during onboarding and by going to **{% my general title="Configuration >> General" %}** .
+The information sent depends on what options you opt-in to. You can opt-in during onboarding and by going to {% my general title="Configuration >> General" %}.
```

Comment:
Not sure why this styling has been removed and how it related to fixing a broken link?

Reply:
It's not related. Sorry.
Markdown linter was giving me a warning about this one (MD036 - Emphasis used instead of a heading)
The bold style here has no effect on the 'My' button anyway.
Now it is consistent with all other 'My' buttons in the documentation.

Maybe this change is tiny enough to leave it in this PR?

Reply:
Yeah, y're right. The button is right below this line. 
The bold hyperlink just looked funny to me:

![image](https://user-images.githubusercontent.com/37924749/136205773-10ded20d-1e93-4845-9a46-2654c1d5431c.png)

Same goes for the `energy` in backticks on the same page:
![image](https://user-images.githubusercontent.com/37924749/136205902-2d3688b7-97ad-4f8a-8a69-51cc5a311845.png)